lib/attn.rb

```diff
@@ -1,16 +1,74 @@
-def attn(method = nil, color = 'yellow')
-	colors = {'red' => '031','green' => '032','yellow' => '033','blue' => '034','magenta' => '035','cyan' => '036'}
-
-	color,method = method,nil if colors.keys.include?(method)
-	color = 'yellow' unless colors.keys.include?(color)
-	
-	unless method
-		print "\033[#{colors[color]}m* * * * * * * * * * * * * * * * * * * * * * * * *\033[0m\n"
+require 'colorize'
+
+$COLORS = ['red','green','yellow','blue','magenta','cyan']
+
+def attn(variable = nil, color = '')
```

Comment:
Since you're adding a handful of new methods at this scope, can you put them into a class or module to avoid risk of naming collision?